homeassistant/components/smartthings/media_player.py

```diff
@@ -270,6 +267,13 @@ def media_artist(self) -> str | None:
     def state(self) -> MediaPlayerState | None:
         """State of the media player."""
         if self.supports_capability(Capability.SWITCH):
+            if not self.supports_capability(Capability.MEDIA_PLAYBACK):
+                if (
+                    self.get_attribute_value(Capability.SWITCH, Attribute.SWITCH)
+                    == "on"
+                ):
+                    return MediaPlayerState.ON
```

Comment:
Is it possible to cover this change with a tesT?

Reply:
I will try to find a new device that has this as well